h2o-docs/src/product/data-science/algo-params/uplift_metric.rst

```diff
@@ -0,0 +1,118 @@
+``uplift_metric``
+-----------------
+
+- Available in: Uplift DRF
+- Hyperparameter: no
```

Comment:
IIMHO it could be hyperparameter

Reply:
Grid search is not supported right now for uplift.

Reply:
ok